src/Jms/YamlConverter.php

```diff
@@ -315,6 +324,9 @@ private function handleClassExtension(&$class, &$data, Type $type, $parentName)
             $property["accessor"]["getter"] = "value";
             $property["accessor"]["setter"] = "value";
             $property["type"] = $alias;
+            if (!$this->useCdata){
```

Comment:
do we need the if condition? cant we just assign it directly?

Reply:
true is the default option for useCdata and jms configuration, so i set it only it's is false.
In this way the jms config files are identical before this commit. And also tests not change